gazebo_ros_control:
  pid_gains:

    joint_1: {p: 100, i: 0, d: 5}
    joint_2: {p: 120, i: 0, d: 8}
    joint_3: {p: 100, i: 0, d: 6}
    joint_4: {p: 50,  i: 0, d: 2}
    joint_5: {p: 10,  i: 0, d: 0.5}
    joint_6: {p: 10,  i: 0, d: 0.5}

    finger_joint: {p: 0, i: 0, d: 0}

    r2_joint_1: {p: 100, i: 0, d: 5}
    r2_joint_2: {p: 120, i: 0, d: 8}
    r2_joint_3: {p: 100, i: 0, d: 6}
    r2_joint_4: {p: 50,  i: 0, d: 2}
    r2_joint_5: {p: 10,  i: 0, d: 0.5}
    r2_joint_6: {p: 10,  i: 0, d: 0.5}

    r2_finger_joint: {p: 0, i: 0, d: 0}

    left_inner_knuckle_joint: {p: 5, i: 0.01, d: 1.0}
    left_inner_finger_joint: {p: 5, i: 0.01, d: 0}
    right_inner_knuckle_joint: {p: 5, i: 0.01, d: 0}
    right_inner_finger_joint: {p: 5, i: 0.01, d: 0}
    right_outer_knuckle_joint: {p: 5, i: 0.01, d: 0}

    r2_left_inner_knuckle_joint:  {p: 0, i: 0, d: 0.0}
    r2_left_inner_finger_joint:   {p: 0, i: 0, d: 0.0}
    r2_right_inner_knuckle_joint: {p: 0, i: 0, d: 0.0}
    r2_right_inner_finger_joint:  {p: 0, i: 0, d: 0.0}
    r2_right_outer_knuckle_joint: {p: 0, i: 0, d: 0.0}
